### ISSUE TITLE
After inserting, saving, and saving again, the "dynamicness" is lost

### ISSUE THREAD

**Issue by Bandit** [nystudio107/richvariables#5]
Steps to reproduce:

1. Insert a variable tag
2. Save
3. Save again
4. Changing the variable doesn't update the tag (in the preview or on the site)

Appears the "preview" of the tag is being saved over the actual tag. Guessing it's something to do with Redactor or the tag previewer?

Craft CMS: `2.6.2983`
Rich Variables: `1.0.5`
Plugins enabled: `Blueprint, Control Panel Nav, Expanded Singles, Minify, Pimp My Matrix, Rich Variables, SEOmatic, Simple Sharing`

**Comment by khalwat** [comment]
Yes, this is an issue with how Craft handles the parsing; they've fixed it for Craft 3, but I'm not sure if they will backport the fixes to Craft 2.x.

**Comment by steveooo9** [comment]
@khalwat Well, this isn't very nice behavior. Would be great if there's a fix, from you or by Craft...

**Comment by khalwat** [comment]
I agree; given the nature of the fix required, it'd be up to P&T to apply the same changes they made to Craft 3 to Craft 2.x. Perhaps file an issue if you're interested in seeing that happening?

https://github.com/craftcms/cms/issues

**Comment by steveooo9** [comment]
See https://github.com/craftcms/cms/issues/2338

**Comment by khalwat** [comment]
https://github.com/craftcms/cms/issues/1643

**Comment by steveooo9** [comment]
@khalwat @Bandit It is marked fixed in newest code. Probably with next Craft 2 release we have the fix. Could anyone also test this if this fix is solid? https://github.com/craftcms/cms/commit/cb2296c4420c84fa8930f48a63f30d5f0f745f00
